extension.ts

```diff
@@ -217,6 +231,13 @@ async function handleKeyEvent(key: string): Promise<void> {
   });
 }
 
+function handleContentChangedFromDisk(document : vscode.TextDocument) : void {
+  _.filter(modeHandlerToEditorIdentity, (modeHandler) => modeHandler.fileName === document.fileName)
```

Comment:
@johnfn I used an arrow function instead, this way renaming variables using vscode won't break anything.